olclient/cli.py

```diff
@@ -65,7 +69,7 @@ def main():
     args = parser.parse_args()
 
     if args.configure:
-        email = args.email
+        email = raw_input("Archive.org Email: ")
```

Comment:
This effectively removes the `--email` arg. Perhaps we should prompt only when `--email` is not present?